documentation/modules/rn-2.6.adoc

```diff
@@ -0,0 +1,93 @@
+// Module included in the following assemblies:
+//
+// * documentation/doc-Release_notes/master.adoc
+
+[id="rn-26_{context}"]
+= {project-full} 2.6
+
+Migrate virtual machines (VMs) from VMware vSphere, {rhv-full}, or {osp} to {virt} with {the-lc} {project-first}.
+
+The release notes describe technical changes, new features and enhancements, and known issues.
+
+[id="technical-changes-26_{context}"]
+== Technical changes
+
+
+This release has the following technical changes:
+
+.Migration from OpenStack moves to being a fully supported feature
+
+In this version of {project-short}, migration using OpenStack source providers graduated from a Technology Preview feature to a fully supported feature.
+
+.Disabling FIPS
+
+EMS enforcement is disabled for migrations with VMware vSphere source providers to enable migrations from versions of vSphere that are supported by {project-short} but do not comply with the 2023 FIPS requirements.
+
```

Comment:
@HagayVider1 FIPS is not disabled; enforcement of it is. Please change the headline accordingly.  Is this new for 2.6? It's in the 2.5 documentation https://access.redhat.com/documentation/en-us/migration_toolkit_for_virtualization/2.5/html-single/installing_and_using_the_migration_toolkit_for_virtualization/index#migrating-vms-web-console

Has something changed. I have not read all the release but this caught my eye as I was looking for the source of the next item.

Reply:
Removed